packaging/docker/build_rpm.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+set -eu
+
+MANAGER_RESOURCES_URL=`curl https://raw.githubusercontent.com/cloudify-cosmo/cloudify-versions/master/packages-urls/manager-single-tar.yaml`
+REMOTE_LOCATION='/root/cloudify-manager-install'
+
+echo "Creating install RPM..."
+docker cp ~/cloudify-manager-install ${CONTAINER_NAME}:${REMOTE_LOCATION}
+docker exec -d ${CONTAINER_NAME} sh -c "systemctl start sshd"
```

Comment:
maybe makes sense to, instead of having 6 `docker exec` calls, have a script in the image and call that with a single exec?